pyartcd/pyartcd/pipelines/report_rhcos.py

```diff
@@ -0,0 +1,138 @@
+from pyartcd.cli import cli, click_coroutine, pass_runtime
+from pyartcd.runtime import Runtime
+
+import click
+import asyncio
+from jenkinsapi.jenkins import Jenkins
+from jenkinsapi.custom_exceptions import NoBuildData
+from subprocess import run
+
+
+class CheckRhcosPipeline:
+    def __init__(self, runtime: Runtime, channel: str):
+        self.runtime = runtime
+        self.channel = channel
+        self.result = dict()
+
+        self.versions = [
+            "4.6",
+            "4.7",
+            "4.8",
+            "4.9",
+            "4.10",
+            "4.11"
+        ]
+
+        self.arches = [
+            "amd64",
+            "s390x",
+            "ppc64le",
+            "arm64"
+        ]
+
+        self.rhcos_urls = {
```

Comment:
The 3 fields above are read-only constants, could be moved outside the class definition (better if renamed in capital letters)